storage/Devices/Encryption.h

```diff
@@ -143,6 +143,24 @@ namespace storage
 	 */
 	void set_pbkdf(const std::string& pbkdf);
 
+	/**
+	 * Get the integrity. For now we only expect AEAD.
+	 *
+	 * Currently only supported for LUKS2 (experimental).
+	 */
+	const std::string& get_integrity() const;
+
+	/**
+	 * Set the integrity. If the integrity is set, this will
```

Comment:
Like other functions (set_pbkdf) it should mention that the value is "Only used during creation.".

Reply:
Done